lib/dockerregistry/types.go

```diff
@@ -50,6 +51,20 @@ type ImageSizeError struct {
 	InvalidImages   map[string]int
 }
 
+// ImageVulnError contains ImageVulnCheck information on images that contain a
+// vulnerability with a severity level at or above the defined threshold.
+type ImageVulnError struct {
+	ImageName             ImageName
```

Comment:
This should also include the Digest, because a single ImageName is by definition ambiguous.

Reply:
Done